packages/platform/src/editor/Editor.tsx

```diff
@@ -262,10 +272,20 @@ const Editor = forwardRef(function Editor<TLogger extends LoggerBasic>(
           {scrRef && onScrRefChange && (
             <ScriptureReferencePlugin
               scrRef={scrRef}
-              onScrRefChange={onScrRefChange}
+              onScrRefChange={localOnScrRefChange}
               viewOptions={viewOptions}
             />
           )}
+          <UsjNodesMenuPlugin
+            trigger="\"
+            scriptureReference={{
+              book: Canon.bookNumberToId(scrRef?.bookNum ?? 0),
+              chapter: scrRef?.chapterNum ?? 0,
+              verse: scrRef?.verseNum ?? 0,
+            }}
+            contextMarker="p"
```

Comment:
@irahopkinson I will need help setting this variable, this is supposed to be the marker selected, so it is a string with the current usfm marker of the node where the cursor is located.

Reply:
Yep, lets talk on your Monday.